### PR TITLE
Create dane-av-enrichments.md

### PR DESCRIPTION
**Pull request by Nanne**
Initial version DANE/CV use case, to be updated with links to use cases that will make use of outcomes from this.

### CODE REVIEW

File: cases/dane-av-enrichments.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
+# Computer vision annotations 'n' enrichments of audiovisual data
+
+## Metadata
+
+* **Status:**  In Progress
+* **Type:** Generic
+* **Work Package**: WP5
+* **Research Coordinators:**  Roeland Ordelman, Nanne van Noord
+* **Coordinators for CLARIAH:**  Nanne van Noord
+* **Participating Institutes:** NISV, UvA
```
Comment (roelandordelman) [inline cases/dane-av-enrichments.md:10]: Eye hier ook noemen?

Comment (Nanne) [reply cases/dane-av-enrichments.md:10]: Voeg ik toe!

**Comment by proycon** [comment]
I assume this is ready for merge now?

**Comment by Nanne** [comment]
Yep!

**Pull request merged**